ert_gui/ertwidgets/analysismodulevariablespanel.py

```diff
@@ -175,8 +176,14 @@ def createSpinBox(
     ):
         spinner = QSpinBox()
         spinner.setMinimumWidth(75)
+
+        # setMaximum() expects a signed 32bit
+        max_int = int(math.pow(2, 31) - 1)
```

Comment:
Can we instead reduce the size of this number:

https://github.com/equinor/ert/blob/4a102a4b520e249bb8be80aaba6679556344daca/ert_gui/ertwidgets/models/analysismodulevariablesmodel.py#L70

And perhaps consider these as well:

https://github.com/equinor/ert/blob/4a102a4b520e249bb8be80aaba6679556344daca/ert_gui/ertwidgets/models/analysismodulevariablesmodel.py#L122

https://github.com/equinor/ert/blob/4a102a4b520e249bb8be80aaba6679556344daca/ert_gui/ertwidgets/models/analysismodulevariablesmodel.py#L130

I don't know if this might change other things as well though.

Reply:
Yea - limiting the input will prevent mistakes in the future. But redusing those numbers should probably be done also